### PR TITLE
remove all usages of "level" to refer to a world

### LINKED ISSUE #241: [discussion] remaining usages of `level` in mappings
I feel like classes such as `LevelInfo` and `LevelProperties` should be refactored to use `World` instead. Consistency! I'm opening this as a discussion because classes that still use level tend to have strings describing them as such.

### PR DESCRIPTION
**Pull request by eristhea**
I hate level why would anyone do this

currently incomplete, it's going to take a lot of searching to finish this

notes:
closes #241  
this pr entirely removes the `net.minecraft.world.level` package because why did it exist at all  
I'm mostly using `WorldSave` to replace the level name  
some decisions on package placement could be improved

### CODE REVIEW

**Review by OroArmor — CHANGES_REQUESTED** [review]
I havent dont a full review, but in QM, level refers to a save, while world is one dimension of that save. Please make sure that you arent overriding level with world when it relates to the whole save

**Comment by eristhea** [comment]
Most replacement is done by replacing level with `WorldSave`
I'm going to pick over this one a lot to make sure everything stays accurate

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/world/storage/WorldSaveStorage.mapping [hunk 1]
```diff
@@ -85,19 +89,19 @@ CLASS net/minecraft/unmapped/C_bpwagnit net/minecraft/world/level/storage/LevelS
...
-	CLASS C_wxntpspm LevelDirectory
+	CLASS C_wxntpspm WorldDirectory
 		FIELD f_iapirmxc path Ljava/nio/file/Path;
 		METHOD <init> (Ljava/nio/file/Path;)V
 			ARG 1 path
 		METHOD equals (Ljava/lang/Object;)Z
 			ARG 1 o
 		METHOD m_awnmrrfa getName ()Ljava/lang/String;
-		METHOD m_bvgoafsy getOldLevelDatPath ()Ljava/nio/file/Path;
+		METHOD m_bvgoafsy getOldWorldDatPath ()Ljava/nio/file/Path;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/storage/WorldSaveStorage.mapping:99]: Is "Dat" short for "Data"? If so, I think it's worth the extra letter for the clarity to just have "Data".

Same for the next two methods ofc.

Comment (eristhea) [reply mappings/net/minecraft/world/storage/WorldSaveStorage.mapping:99]: some parts of minecraft worlds are saved in a `.dat` format, and this is the path to that file

Comment (supersaiyansubtlety) [reply mappings/net/minecraft/world/storage/WorldSaveStorage.mapping:99]: ah ok

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**